.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -0,0 +1,88 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[BUG] <title>"
+labels: ["bug", "needs-triage"]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: SDK Version
+    description: Version of the SDK in use?
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: iOS Version
+    description: Version of iOS in use?
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. In this environment...
+      1. With this config...
+      1. Run '...'
+      1. See error...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Link
+    description: Link to code demonstrating the problem.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Logs
```

```suggestion
    label: Logs / Stacktraces
```
or "Stacktraces / Logs". Stacktraces are more common data form expected with bugs.